stories/utils/generate-widget-stories.js

```diff
@@ -24,16 +24,18 @@ import { storiesOf, Story } from '@storybook/react';
 /**
  * Internal dependencies
  */
-import Widgets from 'googlesitekit-widgets';
+// import Widgets from 'googlesitekit-widgets'; TODO:
```

Comment:
Is it still needed or we can get rid of it?

Reply:
We don't need it, it was a note from a merge conflict I've now fixed.